napari/plugins/_npe2.py

```diff
@@ -420,6 +419,31 @@ def _add_sample(
         plugin_manager._unreg_sample_actions = unreg_sample_actions
 
 
+def _get_multiprovider_submenu(
```

Comment:
I think this name is super unintuitive for what I think this function does which is `maybe_get_submenu` or something like that?

Reply:
Changed to `_get_contrib_parent_menu` and amended docstring